app/v3/controllers/requestParsers/validators/validations/TypeOfBFLossValidation.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v3.controllers.requestParsers.validators.validations
+
+import v3.models.domain.TypeOfBFLoss
+import v3.models.errors.{MtdError, TypeOfLossFormatError}
+
+object TypeOfBFLossValidation {
+  private val typeBFLossTypeNames = List(
+    TypeOfBFLoss.`self-employment`,
+    TypeOfBFLoss.`self-employment-class4`,
+    TypeOfBFLoss.`uk-property-fhl`,
+    TypeOfBFLoss.`uk-property-non-fhl`,
+    TypeOfBFLoss.`foreign-property-fhl-eea`,
+    TypeOfBFLoss.`foreign-property`).map(_.toString)
```

Comment:
Can remove this and `validateForLossClaim` and callers should be using `validate` method in `TypeOfClaimLossValidation` instead.